services/media/service/src/ingest/schemas/ingest-validation-schema.json

```diff
@@ -175,6 +175,21 @@
           }
         }
       }
+    },
+    "localizations": {
+      "type": ["array", "null"],
+      "description": "Localizations for every defined locale.",
```

Comment:
Likely rather "Localizations for any defined locale." or? You don't need to provided all of them (at least it is also possible to send in "null" instead of an array).